tests/test_datasets/test_dataset_functions.py

```diff
@@ -506,6 +506,9 @@ def test__getarff_md5_issue(self):
             "oml:md5_checksum": "abc",
             "oml:url": "https://www.openml.org/data/download/61",
         }
+        n = openml.config.connection_n_retries
```

Comment:
The `OpenMLHashException` leads to retries. Because we know that an `OpenMLHashException` should always be raised (if the unit test passes), we don't want to do this with the default unit test configuration which would try it many times.